tests/trafficmanagement/scripts/request_timeouts.txt

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source ${REPO_ROOT}/content/en/docs/tasks/traffic-management/request-timeouts/snips.sh
+
+# Export HOST, PORT and URL
+export INGRESS_HOST={{ .ingressHostCom }}
+export INGRESS_PORT={{ .ingressPortCom }}
+export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
```

Comment:
Maybe this should also move to bookinfo.txt, so it can be reused by other tests?

Reply:
I also thought about that initially but I'm not sure if we can do that because these are `const` declared in `.go` file that [needs to be attached and passed as a context in `Add` steps](https://github.com/istio/istio.io/pull/6660/files?file-filters%5B%5D=.go&file-filters%5B%5D=.md&file-filters%5B%5D=.txt#diff-084708b4e218661c87edbb7b7d5b9aa3R43) while running a test.